src/campaign/components/ItemCard/ItemNotesModal.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Button,
+  HStack,
+  ModalBody,
+  ModalCloseButton,
+  ModalFooter,
+  ModalHeader,
+  Textarea,
+} from "@chakra-ui/react";
+import { useModal } from "shared";
+import { useEditNotes } from "./useEditNotes";
+
+export const ItemNotes = ({
+  currentNotes,
+  campaignId,
+  itemId,
+}: {
+  currentNotes: string | null;
+  campaignId: string;
+  itemId: string;
+}) => {
+  const { closeModal } = useModal();
+
+  const { notes, setNotes, saveItem, saveActive } = useEditNotes({
+    campaignId,
+    itemId,
+    currentNotes,
+  });
+
+  return (
+    <>
+      <ModalHeader>item notes</ModalHeader>
+      <ModalCloseButton />
+      <ModalBody>
+        <Textarea
+          placeholder="click here to add notes"
+          onChange={(e) => setNotes(e.target.value)}
+          value={notes || ""}
+          minHeight="60vh"
+        />
```

Comment:
Maybe we should have a maxH here too. You can extend the text area higher than the window area at the moment.